middlewares/cachePolicy.js

```diff
@@ -3,7 +3,7 @@ const redis = require('redis');
 const REDISPORT = process.env.REDISPORT || 6379;
 const client = redis.createClient(REDISPORT);
 
-function cache(req,res,next){
+ cache=(req,res,next)=>{
```

Comment:
Revert these changes as these are changes being made outside the domain of the issue you are working on.